File: w1/class/abcz2/abcz2.c
#include <stdio.h>

int main() {
    int value;
    int isZulu = 1;
    
    scanf("%d", &value);
    
    if ( value % 2 == 0 ) {
        printf("alpha\n");
        isZulu = 0;
    }
    if ( value % 3 == 0 ) {
        printf("bravo\n");
        isZulu = 0;
    }
    if ( value % 5 == 0 ) {
        printf("charlie\n");
        isZulu = 0;
    }
    if ( isZulu ) {
        printf("zulu\n");
    }
    
    return 0;
}

File: w4/php_dz/abcz2.php
<?php
    $value = rtrim(fgets(STDIN));
    $isZulu = true;

    if ( $value % 2 == 0 ) {
        $isZulu = false;
        echo "alpha" . PHP_EOL;
    }
    if ( $value % 3 == 0 ) {
        $isZulu = false;
        echo "bravo" . PHP_EOL;
    }
    if ( $value % 5 == 0 ) {
        $isZulu = false;
        echo "charie" . PHP_EOL;
    }
    if ( $isZulu ) {
        echo "zulu" . PHP_EOL;
    }
?>